src/popup/router/components/NodeConnectionStatus.vue

```diff
@@ -49,7 +49,7 @@ export default {
   bottom: env(safe-area-inset-bottom);
   left: 0;
   right: 0;
-  background: variables.$color-bg-3-new;
+  background: variables.$color-bg-3;
```

Comment:
If we want to be sure that the message has the same bg color as the page I suggest to use CSS variable: `--screen-bg-color`.